frontend/components/commons/sidebar-feedback-task/IconButton.vue

```diff
@@ -103,6 +103,13 @@ export default {
       fill: palette(grey, 100);
     }
   }
+  &.custom-expandable :deep(svg) {
+    fill: palette(purple, 200);
+    border: 1px solid palette(purple, 200);
+    border-radius: $border-radius;
+    max-height: 20px;
+    max-width: 20px;
+  }
```

Comment:
move logic for the sidebar outside of the icon component (or the icon component become less generic)